tasks/configure.yml

```diff
@@ -1,6 +1,28 @@
 ---
+
 - name: configure rabbitmq
   template:
     src: rabbitmq.config.j2
     dest: /etc/rabbitmq/rabbitmq.config
   when: rabbitmq_config is defined
+
+- name: rabbitmq full stop
+  include: node_fullstop.yml
+  when: rabbitmq_erlang_cookie is defined
+
+- name: set erlang cookie
+  template:
+    src: erlang.cookie.j2
+    dest: "{{ rabbitmq_erlang_cookie_file }}"
+    owner: rabbitmq
+    group: rabbitmq
+    mode: 0400
+  when: rabbitmq_erlang_cookie is defined
```

Comment:
I like this. It will make it easy to make the erlang_cookie setting optional. You only really want to set it once, during the initial setup and when doing maintenance updates it might be a good idea to leave it out.

This of course may introduce a problem when adding new nodes to the cluster. But I guess that this is a good enough trade-off in this case.

Reply:
yes it is the problem adding new nodes. It should be more parametrized, but still for a new node you still can define that cookie from ansible CLI, so it should be possible.